NOTICE.txt

```diff
@@ -1,6 +1,15 @@
-sonarlint-intellij
+SonarQube for IDE: IntelliJ
 Copyright (C) 2015-2024 SonarSource SA
+All rights reserved.
 mailto:info AT sonarsource DOT com
 
-This product includes software developed at
-SonarSource (http://www.sonarsource.com/).
+This program is distributed with SonarQube analyzers that are subject to specific license terms.
+
+The sonar-csharp-enterprise-plugin and sonar-vbnet-enterprise-plugin
```

Comment:
sonar-cfamily-plugin as well, no?

Reply:
I just copy pasted the legal text 🤷🏻 

I will ask

Reply:
it's true that cfamily plugin is proprietary as well. I guess we missed it because it's not enterprise 😢